app/controllers/spreadsheets_controller.rb

```diff
@@ -81,7 +82,7 @@ def show
             end
           end
 
-          @sheets << { name: name, sheet: sheet, expected: data, actual: hash_tuples(execute(alt_dpps("region = '#{region.gsub('_',' ')}'", 2013, '2013_africa_final'))) }
+          @sheets << { name: name, sheet: sheet, expected: data, actual: hash_tuples(execute(alt_dpps("region = '#{region.gsub('_',' ')}'", latest_analysis.analysis_year, latest_analysis.analysis_name))) }
```

Comment:
This now uses the latest analysis instead of the hard coded one. I'm not sure what this feature is for so this might need to be tweaked if/when it's used.